client/pages/editme/editme.js

```diff
@@ -0,0 +1,100 @@
+//pages/editme/editme.js
+const qcloud = require('wafer2-client-sdk');
+const app = getApp();
+const config = request('../../congig');
+
+Page({
+
+  data: {
+    personal: null,
+    chineseName: null,
+    englishName: null,
+    phoneNumber: null,
+    email: null,
+    birthday: null,
+    btnDisable: true
+  },
+
+  onLoad: function(options) {
+    this.setData({
+      personal: app.userInfo
+    })
+  },
+
+  formSubmit: function (e) { },
+
+  bindBirthdayChange(e) {
+    this.setData({
+      birthday: e.detail.value,
+      btnDisable: 0
+    })
+  },
+  chineseNameInput: function (event) {
+    this.setData({
+      chineseName: event.detail.value,
+      btnDisable: event.detail.value.length === 0
+    });
+  },
+  englishNameInput: function (event) {
+    this.setData({
+      englishName: event.detail.value,
+      btnDisable: event.detail.value.length === 0
+    });
+  },
+  phoneInput: function (event) {
+    this.setData({
+      phoneNumber: event.detail.value,
+      btnDisable: event.detail.value.length === 0
+    });
+  },
+  emailInput: function (event) {
+    this.setData({
+      email: event.detail.value,
+      btnDisable: event.detail.value.length === 0
+    });
+  },
+  formSubmit(e) {
```

Comment:
let's use the same format - either `functionName: function()` or `functionName()`.